webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Overview/index.js

```diff
@@ -19,6 +19,15 @@ const DetailsTab = ({ response, status, hostName }) => {
     <div className="host-details-tab-item details-tab">
       <Grid hasGutter>
         <Slot
+          hostDetails={response}
+          status={status}
+          hostName={hostName}
+          id="host-overview-cards"
+          multi
+        />
+        <Slot
+          deprecated
+          replacedBy="host-overview-cards"
```

Comment:
Added a mechanism to deprecate `<Slot>`s so that this won't be a breaking change. Rather than having your fill not be rendered, you'll instead just get a console warning for now.